### ISSUE TITLE
Browser Sync is Duplicating Click Events

### ISSUE THREAD

**Issue by timdmackey** [bdkjones/CodeKit#522]
**Bug Summary:**
 I am building a css dropdown menu, and I have run into an issue with CodeKit's browser syncing. If multiple instances of the page are open, clicking on the menu dropdown button triggers multiple clicks (equal to the number of open windows), causing the menu to open and then close again in every open window.

**Detailed Description:**
There is no Javascript on the page. For the mobile navigation menu, I am using a form label with an icon and a hidden checkbox, and toggling the dropdown using a css `:checked` selector on the checkbox, eg:

```css
input.nav-expander:checked ~ ul {
  display: block
}
```

The structure of the label toggle is:

```html
<input type="checkbox" id="toggle" />
<label for="toggle">
  <i></i> <!-- Icon -->
</label>
<ul>... <!-- Menu -->
```
**Additional Notes:**
In the process of writing up this report, I noticed that if I click outside of the icon but still inside the label, the click event only fires once. Also, if I change the `<i>` icon element to something else like a `<p>` or a `<span>`, the click event only fires once. I'm not sure what could be at the root of this.

Also, I thought that maybe the form input events may be causing the issue, so I tried disabling syncing of input events in CodeKit. Unfortunately that didn't fix the issue.

**Example:**
Here is the code to a stripped down version of the page where you can see the problem:
https://gist.github.com/timdmackey/fbbe24bbb58c0f6a6fed635d10001ad2

**Comment by bdkjones** [comment]
Believe it or not, CodeKit is doing the right thing. If you create a default `input` of the `checkbox` type, you'll see that browser sync works just fine. The issue you've got is that you're hijacking the input to do something else.

I discovered all sorts of random things when I wrote the browser syncing algorithm in CodeKit and one of those is: "HTML input elements are nuts". Basically, if you click on the `label` of an `input` element, the click is indeed triggered twice: one on the label and once on the input, due to event bubbling. 

Fortunately, there are a couple ways to get around this. See: https://stackoverflow.com/a/30810281/463439
